API/Model/CountryRepository.cs

```diff
@@ -71,7 +82,7 @@ public void DeleteCountry(string code)
                 .Where(c => c.Code == code)
                 .Include(c => c.Data)
                 .ThenInclude(d => d.Points
-                    .Where(p => p.Date.Year == date.Year))
+                .Where(p => p.Date.Year == date.Year))
```

Comment:
Why remove the indent? Was it not clearer before that the Where clause was for datapoints than it is now?